docs/sections.md

```diff
@@ -1,3 +1,8 @@
+---
+layout: page
+nav_exclude: true
+---
+
```

Comment:
@oliverchang 

Adding this heading now excludes `section.md` from the docs site. 

I am not sure what (if any) effect this will have on the current docs site. I thought this was being used in the current docs (it is very similar to what is currently on our docs site) but I just realized that the bit about rate limiting doesn't show up, so maybe it isn't used anywhere? 

This file is removed in its entirety in #1205

Reply:
Hmm, that might break things slightly, but should be OK to just leave it like this for now.

Reply:
Please also see https://github.com/google/osv.dev/pull/1205#issuecomment-1502799424 -- there are some subtleties to completely removing the old docs dir.

Reply:
@oliverchang I think all comments have been resolved for this PR and it should be ready to go. I am happy to adjust anything in #1204 or #1205, or close either of them in favor of other PRs from the devs if necessary.